akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -55,6 +57,27 @@ class HttpExt(private val config: Config)(implicit val system: ActorSystem) exte
 
   private[this] final val DefaultPortForProtocol = -1 // any negative value
 
+  def prefusedConfiguredLayer(settings: ServerSettings, connectionContext: ConnectionContext, log: LoggingAdapter)(implicit mat: Materializer): BidiFlow[HttpResponse, ByteString, ByteString, HttpRequest, NotUsed] = {
```

Comment:
maybe `fuseServerLayer`? Prefused sound like the val where you keep the prefused one